src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -123,26 +104,19 @@ struct GradientVariablesEntropy end
 
 # default to primitive gradient variables
 function CompressibleNavierStokesDiffusion2D(equations::CompressibleEulerEquations2D;
-                                             Reynolds, Prandtl, Mach_freestream,
+                                             viscosity, Prandtl, gas_constant = 287.058,
```

Comment:
If we give a default value for the gas constant, should we consider giving defaults for the other quantities as well (e.g. air under standard conditions)?

Reply:
I thought about this for Prandtl to default to 0.72 and viscosity to default to 0.0. But then I forgot why we defined `prandtl_number()` as a function within the elixirs. I think this was a convenience for the convergence testing and that the initial conditions / source term dispatch on the hyperbolic equations such that within these functions `equations.mu` or `equations.Pr` are unavailable. So setting default values in this way proved difficult.

Reply:
It was for convergence testing - the initial condition is only passed the inviscid equations due to the way the callbacks are structured. You might notice that there aren't equation type annotations for the parabolic elixirs - this is b/c both `CompressibleEulerEquations2D` and `CompressibleNavierStokesEquations2D` are passed in. The same issue occurs with the source. This is only an issue for convergence elixirs, so we used global functions instead to define the parameters. 

I was thinking about a combined equation type to address this in https://github.com/trixi-framework/Trixi.jl/pull/1160, but it's not fully baked IMO.

Reply:
OK, thanks for your feedback. Please see my other comment - if the gas constant has no effect at the moment, I am voting to remove it entirely to avoid confusion (and thus rendering this thread moot).

Reply:
I will remove `R` to make things cleaner. We can kick the question of setting default values for `Pr` and `mu` down the road.